pythoncz_tests/models/github_test.py

```diff
@@ -0,0 +1,432 @@
+import re
+import warnings
+
+import pytest
+import requests
+import responses
+from werkzeug.contrib.cache import NullCache
+
+from pythoncz.models import github as github_module
+from pythoncz_tests.models import github_test_fixtures as fixtures
+
+
+@pytest.fixture()
+def github():
+    """Provides the 'github' module with caching disabled"""
+    original_cache = github_module.cache
+    github_module.cache = NullCache()
+    yield github_module
+    github_module.cache = original_cache
+
+
+@pytest.fixture()
+def requests_mock():
+    """Provides mechanism for faking HTTP responses"""
+    with responses.RequestsMock() as mock:
+        yield mock
+    mock.reset()
+
+
+def test_get_issues_merges_orgs(github, requests_mock):
+    """
+    Tests whether 'get_issues' makes two HTTP requests for each given GitHub
+    organization name and whether both data gets combined in the results
+    """
+    issue_org1 = fixtures.issue()
+    requests_mock.add(
+        responses.POST,
+        'https://api.github.com/graphql',
```

Comment:
I have bad experiences with DRYing tests too much, because it heavily affects readability and clarity of the test case. It's DRYier, but it requires many context switches for the reader to get the whole point of the test. So in tests I'm usually inclining to rather repeat myself. But this might be worth DRYing if the naming is done right, I agree. I'll think about it.